src/store/modules/ftbot/state.ts

```diff
@@ -40,8 +40,10 @@ export interface FtbotStateType {
   selectedPair: string;
   // TODO: type me
   candleData: {};
+  candleDataStatus: 'loading' | 'success' | 'error';
```

Comment:
I'd have created an Enum for this.

That'll avoid having `'loading' | 'success' | 'error'` throughout the code - if we ever have a case of "we need to add one" - it'll be a pain to change throughout the code (and if it's just to fix typescript errors)

Reply:
I initially wanted to put an enum because it just feels right to do so however I'm not sure how.

If you could tell me what's the right place to do it I can put it there.

Reply:
i'm not ... THAT quick :laughing: 
But i see you found a nice place for it anyway.